Languages/en.ini

```diff
@@ -580,6 +580,7 @@ AllowScreenEvalSummary=Allow Screen\nEval Summary
 AllowScreenNameEntry=Allow Screen\nName Entry
 AllowScreenGameOver=Allow Screen\nGame Over
 MusicWheelStyle=MusicWheel Style
+LampsEnabled=Lamps Enabled

```

Comment:
Let's remove the preference altogether and just always have it on for people who have profiles.